server/memphis_handlers_schemas.go

```diff
@@ -113,13 +122,9 @@ func validateSchemaName(schemaName string) error {
 func validateSchemaType(schemaType string) error {
 	invalidTypeErrStr := "unsupported schema type"
 	invalidTypeErr := errors.New(invalidTypeErrStr)
-	invalidSupportTypeErrStr := "avro is not supported at this time"
-	invalidSupportTypeErr := errors.New(invalidSupportTypeErrStr)
 
-	if schemaType == "protobuf" || schemaType == "json" || schemaType == "graphql" {
+	if schemaType == "protobuf" || schemaType == "json" || schemaType == "graphql" || schemaType == "avro" {
```

Comment:
not related to here but we have the "GetFilterDetails" endpoint in which you should also return the Avro type

Reply:
Oh. Yep. missed that one.